### ISSUE TITLE
[Research] WCAG 2.1 Rule: Create rules for Pointer Cancellation - 195

### ISSUE THREAD

**Issue by marcjohlic** [IBMa/equal-access#588]
Determine which rules we can create for detecting [2.5.2 pointer cancellation](https://www.w3.org/WAI/WCAG21/Understanding/pointer-cancellation) issues. See original Box doc from Charu for background:
https://ibm.ent.box.com/file/521554952490

——————
- [x] research previous notes, possible ACT rule(s)
- [ ] general rule description (SME & Dev)
- [ ] testcases written (Dev)
- [ ] rule developed (Dev)
- [ ] message written (SME)
- [ ] help written (SME)

Previous internal issue:  https://github.ibm.com/IBMa/accessibility-web-engine/issues/195

**Comment by marcjohlic** [comment]
From @philljenkins on 10/28/21  
@shunguoy I'm addding the following comment, per our Sprint Planning discussion

See box note on [E2E  196 (WCAG2.1) Pointer Gestures](https://ibm.ent.box.com/notes/839472075014)
- this issue is about Pointer Cancellation, so very related to Gestures
- #196

especially note the 
> **Team discussion results:** 
> **_Not high priority due to the fact that there is no reliable way to detect pointer gesture events, or to detect if there are alternatives to a complex gestures._**
> What to do:  
   1. no need to update the toolkit that already has the requirements for dev/design/test: https://www.ibm.com/able/requirements/requirements/#2_5_1
   2. move the issue to the backlog until we have a better approach in the future.

**Comment by mbgower** [comment]
There are currently no [ACT rules](https://act-rules.github.io/rules/) for 2.5.1

**Comment by mbgower** [comment]
The [published failure technique](https://www.w3.org/WAI/WCAG21/Techniques/failures/F101) lists some potential code we could potentially identify in an inspection of the scripting on a page (my emphasis in bold).

**trigger.addEventListener('mousedown'**, closeDialog);
trigger.addEventListener('touchstart', closeDialog);
trigger.addEventListener('pointerdown', closeDialog);

**Comment by mbgower** [comment]
echnical assessment is that a flag for use of onmousedown and othe potential triggers (touchstart, pointerdown) would lead to over reporting, while failing to catch technology-specific down-event listeners built into libraries, which make up the majority of use cases.
While down-event execution-related issues exist, they are not viewed as common enough that benefit from flagging potentials would outweigh the negative of increased time on task for all teams, resulting from an imperfect check for behaviour based on pointer down-events.

**Comment by mbgower** [comment]
https://ibm.webex.com/webappng/sites/ibm/meeting/postinfo/1eb95dc1264b4b37a573841f1dd46fde_I_213351207811427028?tab=transcripts
